airbyte-integrations/connectors/source-pipedrive/source_pipedrive/manifest.yaml

```diff
@@ -411,3 +411,15 @@ spec:
         examples:
           - "2017-01-25 00:00:00Z"
         type: string
+      num_workers:
+        title: Number of concurrent workers
+        description: Number of concurrent workers to be used in the sync
+        type: integer
+        minimum: 1
```

Comment:
can you add a `default` set to 10. To match the low-code manifest definition

Reply:
added